src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -256,9 +256,11 @@ public class SimpleEvents {
 		.description("Called when a player fills a bucket.")
 				.examples("on player filling a bucket:")
 				.since("1.0");
-		Skript.registerEvent("Throwing of an Egg", SimpleEvent.class, PlayerEggThrowEvent.class, "throw[ing] [of] [an] egg", "[player] egg throw")
-				.description("Called when a player throws an egg. You can just use the <a href='#shoot'>shoot event</a> in most cases, " +
-						"as this event is intended to support changing the hatched mob and its chance to hatch, but Skript does not yet support that.")
+		Skript.registerEvent("Egg Throw", SimpleEvent.class, PlayerEggThrowEvent.class, "throw[ing] [of] [an] egg", "[player] egg throw")
+				.description(
+						"Called when a player throws an egg and it lands. You can just use the <a href='#shoot'>shoot event</a> in most cases."
+						+ " However, this event allows modification of properties like the hatched entity type and the number of entities to hatch."
```

Comment:
Plus goes on line before next line to signify that it continues, old description did this, lets keep it.
```suggestion
						"Called when a player throws an egg and it lands. You can just use the <a href='#shoot'>shoot event</a> in most cases. " +
						"However, this event allows modification of properties like the hatched entity type and the number of entities to hatch."
```

Reply:
Maybe add this to code convention?

Reply:
Sure, it's usually general programming. Been doing lots of JavaScript lately. I'll make a suggestion now.